package.json

```diff
@@ -11,6 +11,7 @@
   "author": "Angel Rodriguez <anxolin@gmail.com> (https://github.com/anxolin)",
   "license": "ISC",
   "dependencies": {
+    "debug": "^4.1.1",
```

Comment:
Shouldn't this be a dev dependency?

Reply:
`yarn install --pure-lockfile --production=true` dkipd frb frprnfrncies, so `require(debug)` would break
If we want `debug` to be used only in development then we'll need to `process.env.SOMETHING && require(debug)` or similar. And we do use it in production with `"start": "DEBUG=ERROR-*,WARN-*,INFO-* node src/bot",`